lib/beaker/dsl/install_utils.rb

```diff
@@ -833,6 +835,18 @@ def install_puppet_from_dmg( host, opts )
         on host, "installer -pkg /Volumes/hiera-#{hiera_ver}/hiera-#{hiera_ver}.pkg -target /"
       end
 
+      def install_puppet_freebsd( host, opts )
+        puppet_ver = opts[:version]
+        facter_ver = opts[:facter_version]
+        hiera_ver = opts[:hiera_version]
+
+        if host['platform'] =~ /freebsd-9/
```

Comment:
Don't you have to install it specifically? The base FreeBSD 9 box I was using only had pkg_ stuff on it